using_images/db_images/mongodb.adoc

```diff
@@ -162,15 +162,19 @@ MongoDB settings can be configured with the following environment variables:
 
 |`*MONGODB_NOPREALLOC*`
 |Disable data file preallocation.
-|true
+|True
```

Comment:
@bparees Do you know whether these are case sensitive? It looks like a spec will accept it init-capped, at least.